src/main/java/uk/gov/hmcts/reform/hmc/client/futurehearing/HearingManagementInterfaceApiClient.java

```diff
@@ -28,4 +31,13 @@ HearingManagementInterfaceResponse requestHearing(@RequestHeader(AUTHORIZATION)
                         @RequestHeader("Request-Created-At") String requestCreatedAt,
                         @RequestHeader("transactionIdHMCTS") UUID transactionIdHMCTS,
                         @RequestBody JsonNode data);
+
+    @PutMapping(value = CASE_URL, consumes = APPLICATION_JSON_VALUE, produces = APPLICATION_JSON_VALUE)
+    HearingManagementInterfaceResponse amendHearing(@PathVariable("cid") String caseListingRequestId,
+                                                    @RequestHeader(AUTHORIZATION) String authorization,
+                                                    @RequestHeader("Source-System") String sourceSystem,
+                                                    @RequestHeader("Destination-System") String destinationSystem,
+                                                    @RequestHeader("Request-Created-At") String requestCreatedAt,
```

Comment:
Not entirely specific to this PR's changes, but if all the HMI requests require these same 4 headers (or a subset of them) we could make a little improvement to automatically add them using a `RequestInterceptor`.